environment.yml

```diff
@@ -0,0 +1,13 @@
+name: cta
+channels:
+  - default
+  - cta-observatory
+  - conda-forge
+dependencies:
+  - python>3.5
+  - ctapipe
```

Comment:
Something I do not understand here: we agreed in #2 that we should depend on the master of ctapipe, while here the dependency on ctapipe is bound to its last release in conda, which can be behind the master.

Reply:
Actually a developer has to take care to test his code with the master version. While a user will install the master version when downloading his code (as its is said in the README [https://github.com/FrancaCassol/nectarchain/tree/setup_environment](https://github.com/FrancaCassol/nectarchain/tree/setup_environment)

Reply:
For a developer, this is clear. But I thought it would be easier for users to just stick with a conda environment, without the need to pip install additional codes. Hence my suggestion in #2 to remove the installation of `ctapipe` from the cta-observatory conda channel, but rather integrate the pip installation in the environment itself.